.pre-commit-config.yaml

```diff
@@ -25,7 +25,7 @@ repos:
   rev: v1.16.0
   hooks:
   - id: setup-cfg-fmt
-    args: ["--min-py3-version", "3.5", "--max-py-version", "3.9"]
+    args: ["--min-py3-version", "3.6"]
```

Comment:
This is not actually needed - it's only there because setup-cfg-fmt can't detect what minimum Python 3 to start at if it detects 2.7+. If it detects 3.6+, there's no ambiguity, so you don't have to tell it manually where you start. One less place to duplicate the supported Python version number.

Reply:
Sure, I'll take it out.